dali/pipeline/operators/decoder/nvjpeg_decoder_cpu.h

```diff
@@ -218,6 +219,42 @@ class nvJPEGDecoderCPUStage : public Operator<CPUBackend> {
     }
   }
 
+ inline uint8_t GetJpegEncoding(const uint8_t* input, size_t size) const {
```

Comment:
Can we place this inside the jpeg.cc?

Reply:
It is supposed to be replaced by the one in next nvjpeg release, but we can have it as a general jpeg utility function